network-api/networkapi/templates/buyersguide/pages/consumer_creepometer_page_2023.html

```diff
@@ -1,69 +1,37 @@
 {% load static %}
 
 {% with wrapper_class="tw-py-[60px] medium:tw-py-[120px]" h1_class="tw-text-[36px] tw-leading-[40px] tw-font-normal medium:tw-text-[48px] medium:tw-leading-[56px] medium:tw-font-semibold tw-font-zilla" h2_class="tw-font-light tw-font-zilla tw-text-[28px] tw-leading-[32px] medium:tw-text-[40px] medium:tw-leading-[48px]" h3_class="tw-text-[24px] tw-leading-[28px] medium:tw-text-[28px] medium:tw-leading-[36px]" h6_class="tw-font-sans tw-text-xs tw-font-normal tw-uppercase tw-tracking-[0.5px] tw-text-gray-60 tw-mb-8" body_large_class="tw-text-[18px] tw-leading-[27px] medium:tw-text-[20px] medium:tw-leading-[30px]" body_small_class="tw-text-sans tw-text-[12px] tw-leading-[16px]" %}
+  <div class="tw-overflow-x-hidden">
```

Comment:
Wrapping this file in `<div class="tw-overflow-x-hidden">` which also introduces whitespace/indentation code changes in this file.